cli/sub_command.go

```diff
@@ -534,29 +552,31 @@ func (c *subCmd) printMsg(msg *nats.Msg, reply *nats.Msg, ctr uint) {
 
 		if info == nil {
 			if msg.Reply != "" {
-				fmt.Printf("[#%d] Received on %q with reply %q\n", ctr, msg.Subject, msg.Reply)
+				fmt.Printf("[#%d]%s Received on %q with reply %q\n", ctr, timeStamp, msg.Subject, msg.Reply)
 			} else {
-				fmt.Printf("[#%d] Received on %q\n", ctr, msg.Subject)
+				fmt.Printf("[#%d]%s Received on %q\n", ctr, timeStamp, msg.Subject)
 			}
 		} else if c.jetStream {
-			fmt.Printf("[#%d] Received JetStream message: stream: %s seq %d / subject: %s / time: %v\n", ctr, info.Stream(), info.StreamSequence(), msg.Subject, info.TimeStamp().Format(time.RFC3339))
+			fmt.Printf("[#%d]%s Received JetStream message: stream: %s seq %d / subject: %s / time: %v\n", ctr, timeStamp, info.Stream(), info.StreamSequence(), msg.Subject, info.TimeStamp().Format(time.RFC3339))
 		} else {
-			fmt.Printf("[#%d] Received JetStream message: consumer: %s > %s / subject: %s / delivered: %d / consumer seq: %d / stream seq: %d\n", ctr, info.Stream(), info.Consumer(), msg.Subject, info.Delivered(), info.ConsumerSequence(), info.StreamSequence())
+			fmt.Printf("[#%d]%s Received JetStream message: consumer: %s > %s / subject: %s / delivered: %d / consumer seq: %d / stream seq: %d\n", ctr, timeStamp, info.Stream(), info.Consumer(), msg.Subject, info.Delivered(), info.ConsumerSequence(), info.StreamSequence())
```

Comment:
Not to me, delta-time can not be confused and even with timestamps you can measure the difference between the time the message was received and the timestamp in the message itself. Also if c.jetStream is false then there is no other timestamp printed so no confusion possible.
But I'm not strongly attached so I can remove it if c.jetStream is true if you want that.

Reply:
Then I'd say lets print it only if its delta, that would be useful but haing timestamp here thats different from the message timestamp (which we arent showing) would definitely be confusing.

Reply:
Ok done

Reply:
I dont follow, this one on line 566 still prints timestamp unconditionally 

And we're duplicating the lines here just for the timestamp.  

Lets just remove timestamp and delta completely from all jetstream related messages that seems the easiest.

Reply:
I removed it from just the JS line that had a (JS) time stamp only (there's no JS data time stamp printed in the case c.jetStream is false).

I  also thing in non-JS mode ( c.jetStream is false) it would look strange to not have the timestamp printed just because the message happens to be a JS message (!).

But if you prefer no timestamps at all for the JS stuff I just did that now.

Reply:
I dont mind it on the jetstream ones if its done as I requested, but if I cant have that I'd rather just not have it at all.

So for example, down in the PR we print a JS ACK subject where we have both the timestamp of the message and still the timestamp.  This is confusing and what I keep asking to remove

Reply:
Ah ok, I had mis-understood what you were asking for then. How about how it is now?

Reply:
Pleasse remove all `timeStamp` from an line anywhere that mentions jetstream.

Reply:
Ok so removed not just if c.jetStream